waspc/cli/src/Wasp/Cli/Command/Db/Reset.hs

```diff
@@ -0,0 +1,36 @@
+module Wasp.Cli.Command.Db.Reset
+  ( reset,
+  )
+where
+
+import Control.Concurrent (newChan)
+import Control.Concurrent.Async (concurrently)
+import Control.Monad.IO.Class (liftIO)
+import StrongPath ((</>))
+import System.Exit (ExitCode (..))
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Common (findWaspProjectRootDirFromCwd)
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import qualified Wasp.Cli.Common as Common
+import qualified Wasp.Generator.DbGenerator.Jobs as DB.Jobs
+import Wasp.Generator.Job.IO (readJobMessagesAndPrintThemPrefixed)
+import qualified Wasp.Message as Msg
+
+reset :: Command ()
+reset = do
```

Comment:
Good point I haven't thought about it!
The thing is, `prisma migrate reset` deletes the whole database and then applies all migrations to it. So once it is done, the migrations should be all set and no need to run migrate-dev.

So what is the best thing to do then? I guess just deleting that file is not enough, because then it will ask user to run migrate-dev which we don't want to do?

Or should we instead re-generate that file?

I see now that migrateDev also calls `  generatePrismaClients projectRootDir
` at its very end. Is that something we should also call here? But why hm, aren't prisma clients generated when we compile the project? And don't they depend on the schema, and not the migrations? Now I wonder why they are generated even at the end of the migrate-dev! If you can help me detangle this stuff that would be very helpful.

Reply:
Talking with Shayne, TODOs for myself:
- I should reset the checksum and then regenerate if all is successful, if not I don't do anything, it remains deleted. But I might not even want to write it, since if it is not there it will just result with deeper check that will anyway generate that file if it succeeds.
- We could also remove it when we do `wasp start db`.
- ask Filip why he put generatePrismaClients at the end of the migrateDev. Should I do it in Wasp reset? Is it connected to prisma reset asking to do generation?